conjure-java-core/src/integrationInput/java/com/palantir/product/SetAlias.java

```diff
@@ -44,4 +46,8 @@ public int hashCode() {
     public static SetAlias of(@Nonnull Set<String> value) {
         return new SetAlias(value);
     }
+
+    public static SetAlias empty() {
```

Comment:
I also considered `of()` rather than `empty()`, matching empty object factory method, but I think this is more descriptive.

Reply:
Yeah, no strong preference myself. `empty()` sounds good!